wrappers/cppopencv/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8.2)
+
+set(THREADS_USE_PTHREADS_WIN32 true)
+find_package(Threads)
+find_package( OpenCV REQUIRED )
+
+include_directories(. ${THREADS_PTHREADS_INCLUDE_DIR})
+
+add_executable( freenect-cppopencv cppopencv.cpp)
```

Comment:
Suggestion - make this conditional on `if (BUILD_EXAMPLES)`.

Reply:
that CMakeLists.txt is included only if `BUILD_CV` is enabled. Are you sure you want a 2nd condition?

Reply:
Ideally yes because this is an example.  But I see the existing cvdemo does not depend on `BUILD_EXAMPLES` so it's just a suggestion.